packages/protocol/test/stability/sortedoracles.ts

```diff
@@ -179,15 +207,42 @@ contract('SortedOracles', (accounts: string[]) => {
         })
       })
 
-      it('should return true if report is expired', async () => {
-        await timeTravel(aReportExpiry, web3)
-        const isReportExpired = await sortedOracles.isOldestReportExpired(aToken)
-        assert.isTrue(isReportExpired[0])
+      describe('using the default expiry', () => {
+        it('should return true if report is expired', async () => {
+          await timeTravel(aReportExpiry, web3)
+          const isReportExpired = await sortedOracles.isOldestReportExpired(aToken)
+          assert.isTrue(isReportExpired[0])
+        })
+
+        it('should return false if report is not expired', async () => {
+          const isReportExpired = await sortedOracles.isOldestReportExpired(aToken)
+          assert.isFalse(isReportExpired[0])
+        })
       })
 
-      it('should return false if report is not expired', async () => {
-        const isReportExpired = await sortedOracles.isOldestReportExpired(aToken)
-        assert.isFalse(isReportExpired[0])
+      describe('when a per token expiry is set, which is grater than the default', () => {
```

Comment:
Would the mirror case, when the new expiry is lower than the original, be useful?